Google Scholar.js

```diff
@@ -887,11 +909,33 @@ ItemFactory.prototype.getAttachments = function (doctype) {
 	attachments = [];
 	for (i = 0, ilen = this.attachmentLinks.length; i < ilen; i += 1) {
 		if (!this.attachmentLinks[i]) continue;
-		attachments.push({
-			title:"Google Scholar Linked " + doctype,
-			url:this.attachmentLinks[i],
-			type:"text/html"
-		});
+		if (!Zotero.Utilities.setMultiField || "string" === typeof this.attachmentLinks[i]) {
+			attachments.push({
+				title:"Google Scholar Linked " + doctype,
+				url:this.attachmentLinks[i],
+				type:"text/html"
+			});
+		} else {
+			// DOM fragment and parent doc
+			var block = this.attachmentLinks[i];
+			var doc = block.ownerDocument;
+
+			// String content (title, url, css)
+			var title = doc.getElementsByTagName("title")[0].textContent;
+			var url = doc.documentURI;
+			var css = "*{margin:0;padding:0;}div.mlz-outer{width: 60em;margin:0 auto;text-align:left;}body{text-align:center;}p{margin-top:0.75em;margin-bottom:0.75em;}div.mlz-link-button a{text-decoration:none;background:#cccccc;color:white;border-radius:1em;font-family:sans;padding:0.2em 0.8em 0.2em 0.8em;}div.mlz-link-button a:hover{background:#bbbbbb;}div.mlz-link-button{margin: 0.7em 0 0.8em 0;}";
+
+			// head element
+			var head = doc.createElement("head");
+			head.innerHTML = '<title>' + title + '</title>';
+			head.innerHTML += '<style type="text/css">' + css + '</style>'; 
+
+			var attachmentdoc = Zotero.Utilities.composeDoc(doc, head, block);
+			attachments.push({title:"Google Scholar: "+doctype, type:"text/html", document:attachmentdoc});
```

Comment:
(1) Can we make the title the same across attachments via URL or via this (unless they're actually different content-wise) and (2) `type:"text/html"` is superfluous here because you're using `document` property.